packages/js/data/src/onboarding/selectors.ts

```diff
@@ -64,7 +65,7 @@ export const getTask = (
 
 export const getPaymentGatewaySuggestions = (
 	state: OnboardingState
-): PaymentMethodsState[] => {
+): Plugin[] => {
```

Comment:
I found `PaymentGatewaySuggestions` is actually the Plugin type. So `PaymentMethodsState` is redundant.